app/src/main/java/ch/epfl/qedit/model/Quiz.java

```diff
@@ -144,14 +138,6 @@ private void checkState() {
                 throw new IllegalStateException("Builder already build once.");
             }
         }
-
-        /** Checks that the question has the same treasure-hunt settings as the quiz */
-        private void checkQuestion(Question question) {
-            boolean questionTreasureHunt = question.getLocation() != null;
-            if (questionTreasureHunt != treasureHunt)
-                throw new IllegalArgumentException(
-                        "Cannot mix treasure-hunt and non treasure-hunt questions");
-        }
```

Comment:
Why do we get ride of this check ?

Reply:
we could check radius != -1 no ?

Reply:
I assumed that nobody would create invalid quizzes. But you are right, checking the radius might make the whole thing a bit safer, although not perfect.